jenkins_dc/Jenkinsfile

```diff
@@ -85,5 +76,18 @@ pipeline{
         always{
             sh 'docker image prune -a -f'
         }
+        success {
+        emailext body: 'The Convert service build was executed successfully',
+                 subject: 'Success results for the "Converter service"',
```

Comment:
Use environment variables to improve your email notification. always {}
Pipeline for ${BUILD_NUMBER} - has been executed - Status: ${BUILD_STATUS}
The Convert service build was executed ${BUILD_STATUS} 
This is the log link